app/bundles/FormBundle/Validator/Constraint/PhoneNumberConstraintValidator.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony2 PhoneNumberBundle.
+ *
+ * (c) University of Cambridge
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Mautic\FormBundle\Validator\Constraint;
+
+use libphonenumber\NumberParseException;
+use libphonenumber\PhoneNumber as PhoneNumberObject;
+use libphonenumber\PhoneNumberFormat;
+use libphonenumber\PhoneNumberType;
+use libphonenumber\PhoneNumberUtil;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * Phone number validator.
+ */
+class PhoneNumberConstraintValidator extends ConstraintValidator
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (null === $value || '' === $value) {
+            return;
+        }
+
+        if (!is_scalar($value) && !(is_object($value) && method_exists($value, '__toString'))) {
+            throw new UnexpectedTypeException($value, 'string');
+        }
+
+        $phoneUtil = PhoneNumberUtil::getInstance();
+
+        if (false === $value instanceof PhoneNumberObject) {
+            $value = (string) $value;
+
+            try {
+                $phoneNumber = $phoneUtil->parse($value, $constraint->defaultRegion);
+            } catch (NumberParseException $e) {
+                $this->addViolation($value, $constraint);
+
+                return;
+            }
+        } else {
+            $phoneNumber = $value;
+            $value       = $phoneUtil->format($phoneNumber, PhoneNumberFormat::INTERNATIONAL);
+        }
+
+        if (false === $phoneUtil->isValidNumber($phoneNumber)) {
+            $this->addViolation($value, $constraint);
+
+            return;
+        }
+
+        switch ($constraint->getType()) {
+            case PhoneNumber::FIXED_LINE:
```

Comment:
This causes error. In use statement is "use libphonenumber\PhoneNumber as PhoneNumberObject;" so it needs to be used with PhoneNumberObject alias